<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rajadurai-PC" failures="1" tests="4" name="Test" time="87.174" errors="0" timestamp="2024-09-30T15:35:24 IST">
  <testcase classname="testCases.Automation_TC_002" name="verifyLogIn" time="16.656">
    <skipped/>
  </testcase> <!-- verifyLogIn -->
  <testcase classname="testCases.Automation_TC_002" name="verifyLogIn" time="17.039">
    <skipped/>
  </testcase> <!-- verifyLogIn -->
  <testcase classname="testCases.Automation_TC_002" name="verifyLogIn" time="16.654">
    <skipped/>
  </testcase> <!-- verifyLogIn -->
  <testcase classname="testCases.Automation_TC_002" name="verifyLogIn" time="17.579">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.Automation_TC_002.verifyLogIn(Automation_TC_002.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogIn -->
</testsuite> <!-- Test -->
